app/src/main/java/ar/com/wolox/android/cookbook/twitterlogin/TwitterLoginRecipeFragment.kt

```diff
@@ -69,6 +69,15 @@ class TwitterLoginRecipeFragment : WolmoFragment<TwitterLoginRecipePresenter>(),
 
     override fun toggleLoginButtonState(status: Boolean) {
         vTwitterLoginBtn.isClickable = status
+        vApiTwitterLoginBtn.isEnabled = status
+        vLogoutBtn.isEnabled = !status
+        if (status) {
+            vLoginTwitterSignInInstructions.visibility = View.VISIBLE
+            vLoginTwitterLogoutInstructions.visibility = View.GONE
+        } else {
+            vLoginTwitterSignInInstructions.visibility = View.GONE
+            vLoginTwitterLogoutInstructions.visibility = View.VISIBLE
```

Comment:
same as above. Also, here you can use the value of `status` to avoid the if:
```
vLoginTwitterSignInInstructions.toggleVisibility(status)
vLoginTwitterSignInInstructions.toggleVisibility(!status)
```
So, maybe you can rename status to make it more descriptible when calling toggleVisibility

Reply:
same as above